test/common/http/conn_manager_impl_test.cc

```diff
@@ -932,6 +932,11 @@ TEST_F(HttpConnectionManagerImplTest,
 }
 
 TEST_F(HttpConnectionManagerImplTest, TestAccessLog) {
+  constexpr char local_address[] = "0.0.0.0";
```

Comment:
Ideally we also have a more specific unit test on `ConnectionManagerImpl::ActiveStream::ActiveStream`, i.e. HCM, that shows this behavior. If there's a reasonable way to add this without too many gymnastics, that could be great.